src/prefect/blocks/notifications.py

```diff
@@ -128,3 +139,108 @@ class MicrosoftTeamsWebhook(AppriseNotificationBlock):
         description="The Teams incoming webhook URL used to send notifications.",
         example="https://your-org.webhook.office.com/webhookb2/XXX/IncomingWebhook/YYY/ZZZ",
     )
+
+
+class PagerDutyWebHook(AbstractAppriseNotificationBlock):
+    """
+    Enables sending notifications via a provided PagerDuty webhook.
+    See [Apprise notify_pagerduty docs](https://github.com/caronc/apprise/wiki/Notify_pagerduty)
+    for more info on formatting the URL.
+
+    Examples:
+        Load a saved PagerDuty webhook and send a message:
+        ```python
+        from prefect.blocks.notifications import PagerDutyWebHook
+        pagerduty_webhook_block = PagerDutyWebHook.load("BLOCK_NAME")
+        pagerduty_webhook_block.notify("Hello from Prefect!")
+        ```
+    """
+
+    _description = "Enables sending notifications via a provided PagerDuty webhook."
+
+    _block_type_name = "Pager Duty Webhook"
+    _block_type_slug = "pager-duty-webhook"
+    _logo_url = "https://images.ctfassets.net/gm98wzqotmnx/6FHJ4Lcozjfl1yDPxCvQDT/c2f6bdf47327271c068284897527f3da/PagerDuty-Logo.wine.png?h=250"
+
+    # The default cannot be prefect_default because NotifyPagerDuty's
```

Comment:
What's "prefect_default"?

Reply:
https://github.com/PrefectHQ/prefect/pull/7534/files#diff-0da1e167514eabebc23630ecf4ccc47b12cbb17567ac00a1e9723879a8715caeL28-L36
```
    Attributes:
        DEFAULT: A plain notification that does not insert any notification type images.
```

I think some plugins do not support images, thus Ryan added a prefect_default.

Reply:
Hm okay. Weird that severity levels are being used for notification types.